apply/default_applier.go

```diff
@@ -260,5 +290,7 @@ func NewDefaultApplier(cluster *v1.Cluster) (Interface, error) {
 		FileSystem:     fs,
 		Guest:          gs,
 		Config:         config.NewConfiguration(cluster.Name),
+		Plugin:         plugin.Config(cluster.Name),
```

Comment:
Like other modules, plugins,err := plugin.NewPlugins()

```
type plugins interface {
     Dump()
     Run()
}
```

We may confused about the differece of Plugin and Plugins, I think applier don't need care about Plugin, just using Plugins is fine.

Reply:
The modified